src/pages/styles.module.css

```
@@ -35,3 +37,38 @@
   height: 200px;
   width: 200px;
 }
+
+/**
+* For table of content items 
+*/
+.TOC {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-self: center;
+  margin: 1.5vh 0;
+  padding: 0;
+  
+  ul {
```

This still needs the `>`. It tells css to only style the immediate child of the .TOC class. Using it without the `>` will change the styling of any unordered list inside the ` .TOC` class

```suggestion
  > ul {
```